README.md

```diff
@@ -26,6 +26,15 @@ A user can add a question to rocket fuel directly in the ui, and then wait for a
 
 Feel free to contribute to rocket fuel. Create a issue, when the issue is accepted, create a fork and create a pull request. 
 
+#### Prerequisites
+
+* Authentication requires you to set up a client in the Google Developer console.
```

Comment:
Are there any particular settings in the devconsole you need to set, or is it just "Create auth things" and it will work?

Reply:
Fixed!